platform/vs/tests/breakout/test_breakout_cli.py

```diff
@@ -0,0 +1,166 @@
+from swsscommon import swsscommon
+import time
+import os
+import json
+import ast
+import pytest
+import collections
+
+@pytest.mark.usefixtures('dpb_setup_fixture')
+class TestBreakoutCli(object):
+    def setup_db(self, dvs):
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    def read_Json(self, dvs):
+        test_dir = os.path.dirname(os.path.realpath(__file__))
+        sample_output_file = os.path.join(test_dir, 'sample_output', 'sample_new_port_config.json')
+        fh = open(sample_output_file, 'rb')
+        fh_data = json.load(fh)
+        fh.close()
```

Comment:
```suggestion
        with open(sample_output_file, 'rb') as fh:
            fh_data = json.load(fh)
```

This is a safer way to access the file in case something happens and the open fails or the load fails and the close statement is never reached. :)

Reply:
Addressed as requested.